### PR TITLE
Update training_with_built_in_methods.md

### PR DESCRIPTION
**Pull request by jvishnuvardhan**
Previously, `loss` and `sparse_categorical_accuracy` from `history` object were not matching with the values obtained during `model.fit`. 

I updated values of `loss` and `sparse_categorical_accuracy` to match with the values obtained during training.

### CODE REVIEW

**Comment by fchollet** [comment]
These values are going to be  different every time we run the notebook, no?

**Comment by jvishnuvardhan** [comment]
> These values are going to be different every time we run the notebook, no?

Correct. Values will be different every time we run the code. But, the values shown after `model.fit` shouldn't be different when compared to the values under `model.history` object.

The page on Keras website shows the following results after `model.fit`

Fit model on training data
```
Epoch 1/2
782/782 [==============================] - 2s 2ms/step - loss: 0.5776 - sparse_categorical_accuracy: 0.8435 - val_loss: 0.1810 - val_sparse_categorical_accuracy: 0.9475
Epoch 2/2
782/782 [==============================] - 1s 978us/step - loss: 0.1679 - sparse_categorical_accuracy: 0.9511 - val_loss: 0.1637 - val_sparse_categorical_accuracy: 0.9529
```

Then, the history object shows different `loss` as shown below. 

```
{'loss': [0.3402276635169983, 0.15610544383525848],
 'sparse_categorical_accuracy': [0.9048200249671936, 0.9537400007247925],
 'val_loss': [0.1809607595205307, 0.16366209089756012],
 'val_sparse_categorical_accuracy': [0.9474999904632568, 0.9528999924659729]}


```

But, `loss` in `history` object should match with the `loss` shown after `model.fit`.

**Review by fchollet — APPROVED** [review]
Got it. LGTM, thanks!
